eos-update-flatpak-repos

```diff
@@ -130,6 +130,33 @@ def _remove_remotes():
                                    '--system', '--force', name])
 
 
+# dict of remote -> URLs to migrate
+REMOTES_TO_MIGRATE = {
+    "eos-runtimes": {
+        "old-url": "https://ostree.endlessm.com/ostree/eos",
+        "new-url": "https://ostree.endlessm.com/ostree/eos-runtimes",
+    },
+}
+
+
+def _migrate_remotes():
+    insts = Flatpak.get_system_installations()
+    inst = insts[0]
+
+    for name, urls in REMOTES_TO_MIGRATE.items():
+        remote = _flatpak_inst_get_remote(inst, name)
+        if remote and remote.get_url() == urls["old-url"]:
+            logging.info("Migrating remote {} from {} to {}"
+                         .format(name, urls["old-url"], urls["new-url"]))
+
+            subprocess.check_call(["flatpak", "remote-modify",
```

Comment:
Why use the CLI? `_fix_remote_urls` (lower down this file) uses the libflatpak API to do something rather similar.

Reply:
Fine by me. Most of the other code uses the CLI, but I prefer the API.